core/pom.xml

```diff
@@ -32,10 +32,16 @@
 	<packaging>jar</packaging>
 
     <dependencies>
+        <dependency>
+            <groupId>org.locationtech.jts</groupId>
+            <artifactId>jts-core</artifactId>
+            <version>1.16.2-SNAPSHOT</version>
```

Comment:
@netanel246 Note that, this dependency is actually the JTS submodule of the git repo, rather than a standard JTS Maven release. It was compiled by Sedona "mvn install" and automatically added to Sedona, locally. The "SNAPSHOT" comes from here: https://github.com/jiayuasu/jts/blob/1.16.x/modules/core/pom.xml

The git submodule is Sedona Maven submodule: https://github.com/apache/incubator-sedona/blob/jtsplus/pom.xml#L12

Reply:
Alright, Thanks :+1: